src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -503,7 +574,22 @@ private void TargetStarted(object sender, TargetStartedEventArgs e)
             project.Stopwatch.Start();
 
             string projectFile = Path.GetFileNameWithoutExtension(e.ProjectFile);
-            NodeStatus nodeStatus = new(projectFile, project.TargetFramework, e.TargetName, project.Stopwatch);
+
+            var isTestTarget = e.TargetName == _testTarget;
+
+            var targetName = isTestTarget ? "Testing" : e.TargetName;
```

Comment:
This avoid showing ugly internal name, until first test result arrives.  Targets are not normally localized, so I also don't localize this.